src/config/settings.ts

```diff
@@ -445,6 +445,7 @@ const settings: Settings = {
       },
     },
     priceFreshness: parseInt(process.env.PRICE_FRESHNESS || process.env.KAIKO_FRESHNESS || '3600', 10),
+    liquidityFreshness: parseInt(process.env.LIQUIDITY_FRESHNESS || String(getDayInMillisecond(1)), 10),
```

Comment:
this should go to dex section, this is dex related setup

Reply:
I moved to dexes, but I didn't add to each chain. We probably want the same value for all of them, or am I wrong?

Reply:
In fact I needed to move inside the chain and protocol. We use the Object.keys in some parts of code and it turns difficult to handle it in the same level as chain or protocol